cmd/nsm-init/nsm-init.go

```diff
@@ -79,25 +204,10 @@ func main() {
 	logrus.Infof("nsm client: client api %+v", nsmClient)
 	// Init related activities start here
 
-	currentNamespace, err := netns.Get()
-	if err != nil {
-		logrus.Errorf("nsm client: failure to get pod's namespace with error: %+v", err)
+	if err := applyRequiredConfig(ns); err != nil {
+		logrus.Infof("nsm client: initialization failed, exiting...", err)
```

Comment:
We should use fatal, so that the log level is recorded properly.

Reply:
done